automation/test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+# CI considerations: $TARGET is used by the jenkins build, to distinguish what to test
+# Currently considered $TARGET values:
+#     kubernetes-release: Runs all functional tests on a release kubernetes setup
+#     openshift-release: Runs all functional tests on a release openshift setup
+
+set -ex
+
+export WORKSPACE="${WORKSPACE:-$PWD}"
+
+if [[ $TARGET =~ openshift-.* ]]; then
+  export KUBEVIRT_PROVIDER="os-3.10.0"
+elif [[ $TARGET =~ .*-1.10.4-.* ]]; then
+  export KUBEVIRT_PROVIDER="k8s-1.10.4"
+fi
+
+export KUBEVIRT_NUM_NODES=2
+
+kubectl() { cluster/kubectl.sh "$@"; }
+
+export NAMESPACE="${NAMESPACE:-kube-system}"
+
+make cluster-down
+make cluster-up
+
+# Wait for nodes to become ready
+set +e
+kubectl get nodes --no-headers
+kubectl_rc=$?
+while [ $kubectl_rc -ne 0 ] || [ -n "$(kubectl get nodes --no-headers | grep NotReady)" ]; do
+    echo "Waiting for all nodes to become ready ..."
+    kubectl get nodes --no-headers
+    kubectl_rc=$?
+    sleep 10
+done
+set -e
+
+echo "Nodes are ready:"
+kubectl get nodes
+
+make cluster-sync
+
+kubectl version
+
+ginko_params="--ginkgo.noColor --junit-output=$WORKSPACE/junit.xml"
```

Comment:
AFAIK stdci job is expects junit results to match this pattern `exported-artifacts/*.junit.xml`

Reply:
Copied that line from the kubevirt test.sh (in fact copied that file, and removed the bits we don't need)

Reply:
That might be reason why results for kubevirt's PRs are not available?

Reply:
Maybe, I am not entirely sure which results you are talking about? You have an example where it does work?

Reply:
For example here http://jenkins.ovirt.org/job/kubevirt_kubevirt-ansible_standard-check-pr/1160/testReport/(root)/

When I check some of kubevirt's PR there are no results for example here http://jenkins.ovirt.org/job/kubevirt_kubevirt_standard-check-pr/1715/